app/schemas/pregnancy-center.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const Joi = require('joi')
+const phoneValidator = require('joi-phone-validator')
+
+const pointSchemaJoi = Joi.object().keys({
+	_id: Joi.string(),
+	type: Joi.string().valid('Point').required(),
+	coordinates: Joi.array().ordered(
+		Joi.number().max(-66).min(-128).required(), // general continental US longitude parameters
+		Joi.number().min(23).max(50).required() // general continental US latitude parameters, to keep from accidently switching lat, lng
+	).min(2).max(2)
+})
+
+const addressSchemaJoi = Joi.object().keys({
+	_id: Joi.string(),
+	city: Joi.string(),
+	googlePlaceId: Joi.string(), // we can store the google place ID according to TOS
+	line1: Joi.string(),
+	line2: Joi.string(),
+	location: pointSchemaJoi,
+	state: Joi.string(),
+	zip: Joi.string(),
+})
+
+const queryableDayHoursSchemaJoi = Joi.object().keys({
+	open: Joi.number().min(0).max(86400), // number of seconds since 00:00:00
+	close: Joi.number().min(0).max(86400) // number of seconds since 00:00:00
```

Comment:
This is totally optional, but an idea to make a developer understand what that number means (rather than in the comments is to do something like `.max(60 * 60 * 24) so you can quickly look at that and know it's 60 seconds * 60 minutes * 24 hours. And the way the JS interpreter understands it is essentially the same.